### DISCUSSION TITLE
Group Assignment #3

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#130] [Assignments]
# Survey
## Due Thursday 4/29

Design a survey that will focus on the community voices for your project.

You may also revise your memo to ensure that you focus on specific community members or group. You can check this [Google Sheet for feedback from other students](https://docs.google.com/spreadsheets/d/1B5QTtB3iDFuOXT3z6kv1Dwr1eNku9AeVAHF55YEZXFs/edit#gid=1767595116).

## Submission:
1. Submit your survey using this Google Form:
https://forms.gle/5aPsriPe1iCou6qG8

2. For revised memos, comment in this thread with the link.

**Comment by rachan2023** [comment]
https://docs.google.com/document/d/12e01HPhnCDphWnRJ5EHmwt0Eu-FqUv5k-ukRLZbkv2E/edit?usp=sharing

**Comment by epham36** [comment]
Group 6: Jason, Emily, Jess

https://docs.google.com/document/d/1Er8Jq6M4fRsptmrNRLiWepqMFx1uWburt91TsRSp5G0/edit?usp=sharing

**Comment by annyrviloria** [comment]
Group 1: Migrant Justice / Justicia Migratoria

https://docs.google.com/document/d/1DH6YNUFHU2uSeaVnlBn4IzXZT_FO6lTki5XhPlSB_BA/edit?usp=sharing

**Comment by Wny-Duong** [comment]
Group 4: Ericko Halim, Latisha Sumardy, Wilny Duong
https://docs.google.com/document/d/1rWZ6G-Pj3Jj16WsD7iaiicDROEX7VdmYcXNI4Tlb9cs/edit?usp=sharing

**Comment by brillianz** [comment]
https://docs.google.com/document/d/1GKSrchk_-NSHU-dDEOb8n-9Ya_Nxqn8qWPena-MbM0o/edit?usp=sharing
Group #7: Ray, Sam, and Brian.
